src/buckets/components/lineProtocol/EnterManuallyButtons.tsx

```diff
@@ -37,7 +42,11 @@ const EnterManuallyButtons: FC = () => {
 
   const selectedBucket =
     useSelector((state: AppState) =>
-      getByID<Bucket>(state, ResourceType.Buckets, bucketID)
+      getByID<Bucket>(
+        state,
+        ResourceType.Buckets,
+        bucketID ?? chosenUploadBucket?.id
```

Comment:
Not sure if the bucket ID JS relevant anymore after this change. I’d double check

Reply:
It is in this component, since this isn't the only place this component is utilized

Reply:
I think it might be now though.

Reply:
yep, ur right - now it is... my bad. I removed it!